src/main/java/ch/njol/skript/expressions/ExprJoinSplit.java

```diff
@@ -42,16 +42,23 @@
 @Name("Join & Split")
 @Description("Joins several texts with a common delimiter (e.g. \", \"), or splits a text into multiple texts at a given delimiter.")
 @Examples({"message \"Online players: %join all players with \"\" | \"\"%\" # %all players% would use the default \"x, y, and z\"",
-		"set {_s::*} to the string argument split at \",\""})
-@Since("2.1")
+	"set {_s::*} to the string argument split at \",\""})

```

Comment:
This was an unnecessary change
```suggestion
		"set {_s::*} to the string argument split at \",\""})

```